.gitignore

```diff
@@ -9,6 +9,8 @@ node_modules
 /util/python/python_lib
 /pip_test
 /_python_build
+**/tools/git/gen
```

Comment:
For some reason the `gen` folder is being generated on my machine, and it is pointing to my `.git` folder